.github/workflows/build_documentation.yml

```
@@ -76,41 +77,24 @@ jobs:
           git config --global user.name "Hugging Face Doc Builder"
           git config --global user.email docs@huggingface.co
           
-          cd doc-build
-          git pull origin main
-          cd ..
-
-          cd notebooks
-          git pull origin master
-          cd ..
-
       - name: Make documentation
         run: |
           cd doc-builder &&
-          doc-builder build transformers ../transformers/docs/source --build_dir ../doc-build --notebook_dir notebooks/transformers_doc --clean --html &&
+          doc-builder build transformers ../transformers/docs/source --build_dir ../build_dir --notebook_dir ../notebooks_dir --clean --html &&
           cd ..
         env:
           NODE_OPTIONS: --max-old-space-size=6656
 
       - name: Push to repositories
         run: |
           cd doc-build &&
-          if [[ `git status --porcelain` ]]; then 
-            git add . &&
-            git stash && git pull && git stash apply &&
-            git commit -m "Updated with commit ${{ github.sha }} \n\nSee: https://github.com/huggingface/transformers/commit/${{ github.sha }}" &&
-            git push origin main
-          else
-            echo "No diff in the documentation."
-          fi &&
+          git pull &&
+          cp -r ../builder_dir/transformers . &&
+          git status &&
           cd .. &&
 
           cd notebooks &&
-          if [[ `git status --porcelain` ]]; then
-            git add transformers_doc &&
-            git commit -m "Updated Transformer doc notebooks with commit ${{ github.sha }} \n\nSee: https://github.com/huggingface/transformers/commit/${{ github.sha }}" &&
-            git push origin master
-          else
-            echo "No diff in the notebooks."
-          fi &&
+          git pull &&
+          cp -r ../notebooks_dir/. transformers_doc &&
```

```suggestion
          cp -r ../notebooks_dir ./transformers_doc &&
```
was `../notebooks_dir/.` intended?

---

Yes, to copy everyhing inside it to `transformers_doc/

---

got it